torchtext/prototype/models/t5/bundler.py

```diff
@@ -0,0 +1,152 @@
+import logging
+from dataclasses import dataclass
+from typing import Any, Dict, Optional, Union
+from urllib.parse import urljoin
+
+import torch
+from torchtext import _TEXT_BUCKET
+from torchtext._download_hooks import load_state_dict_from_url
+
+from .model import T5Conf, T5Model
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class T5Bundle:
+    """T5Bundle(_config: torchtext.prototype.models.T5Conf, _path: Optional[str] = None)
+
+    Example - Pretrained base t5 encoder
+        >>> import torch, torchtext
+        >>> t5_encoder_base = torchtext.prototype.models.T5_BASE_ENCODER
+        >>> model = t5_encoder_base.get_model()
+        >>> model_input = torch.tensor([[1,2,3,4,5,6],[7,8,9,0,0,0]])
+        >>> output = model(model_input)['encoder_output']
+        >>> output.shape
+        torch.Size([2, 6, 768])
+
+    Example - Pretrained base t5 model
+        >>> import torch, torchtext
+        >>> t5_base = torchtext.prototype.models.T5_BASE
+        >>> model = t5_base.get_model()
+        >>> encoder_input = torch.tensor([[1,2,3,4,5,6],[7,8,9,0,0,0]])
+        >>> decoder_input = torch.tensor([[0],[0]])
+        >>> output = model(encoder_input, decoder_input)['decoder_output']
+        >>> output.shape
+        torch.Size([2, 1, 768])
+
+    Example - User-specified configuration and checkpoint
+        >>> from torchtext.prototype.models import T5Conf, T5Bundle
+        >>> model_weights_path = "https://download.pytorch.org/models/text/t5.base.encoder.pt"
+        >>> encoder_conf = T5Conf(encoder_only=True)
+        >>> model = T5Bundle.build_model(config=encoder_conf, checkpoint=model_weights_path)
+    """
+
+    _config: T5Conf
+    _path: Optional[str] = None
+
+    def get_model(
+        self,
+        *,
```

Comment:
I saw for the Roberta bundler that `get_model` and `build_model` accepted `*` as an input argument, however I'm not too sure why that is since I don't see it being used in the methods. I kept them here for now, but want to verify their purpose before removing.

Reply:
This will enforce user to  pass all arguments after `*` using keywords only in function call. They can no longer be passed as positional arguments. This may be useful when changing API in the future since the arguments were passed using their names, changing their positions would not cause breakage in existing code base.